### PR TITLE
feed minor change

### PR DESCRIPTION
**Pull request by tattax**
## About The Pull Request

makes it so levels actually give you +1 second on your feed sleep instead of 0.1 seconds

## Why It's Good For The Game

more blood on bloodsuckers

:cl:
add: readds silent / non silent feeding differences
balance: feed sleep now gains +1 second per level instead of 0.1
/:cl:

### CODE REVIEW

File: fulp_modules/features/antagonists/bloodsuckers/code/powers/feed.dm [hunk 1]
```diff
@@ -25,6 +25,8 @@
 	var/warning_target_bloodvol = BLOOD_VOLUME_MAX_LETHAL
 	///Reference to the target we've fed off of
 	var/datum/weakref/target_ref
+	///Are we feeding with aggresive grab or not?
+	var/silent_feed = TRUE
```
Comment (JohnFulpWillard) [inline fulp_modules/features/antagonists/bloodsuckers/code/powers/feed.dm:29]: Comment says it's whether the feed is aggressive, but aggressive shouldn't be silent

File: fulp_modules/features/antagonists/bloodsuckers/code/powers/feed.dm [hunk 2]
```diff
@@ -131,7 +133,28 @@
 	var/mob/living/user = owner
 	var/mob/living/feed_target = target_ref.resolve()
 	if(!ContinueActive(user, feed_target))
-		owner.balloon_alert(owner, "interrupted!")
+		if(silent_feed)
```
Comment (JohnFulpWillard) [inline fulp_modules/features/antagonists/bloodsuckers/code/powers/feed.dm:136]: When is this decided? It's not always silent

Comment (tattax) [reply fulp_modules/features/antagonists/bloodsuckers/code/powers/feed.dm:136]: it got lost in translation

File: fulp_modules/features/antagonists/bloodsuckers/code/powers/feed.dm [hunk 3]
```diff
@@ -92,13 +94,13 @@
 		return
 	if(owner.pulling == feed_target && owner.grab_state >= GRAB_AGGRESSIVE)
 		if(!IS_BLOODSUCKER(feed_target) && !IS_VASSAL(feed_target) && !IS_MONSTERHUNTER(feed_target))
-			feed_target.Unconscious((5 SECONDS + level_current))
+			feed_target.Unconscious((5 + level_current) SECONDS)
 		if(!feed_target.density)
 			feed_target.Move(owner.loc)
-	if(owner.pulling == feed_target && owner.grab_state >= GRAB_AGGRESSIVE)
```
Comment (JohnFulpWillard) [inline fulp_modules/features/antagonists/bloodsuckers/code/powers/feed.dm:98]: Why're we removing this?

File: fulp_modules/features/antagonists/bloodsuckers/code/powers/feed.dm [hunk 4]
```diff
@@ -92,13 +94,13 @@
 		return
 	if(owner.pulling == feed_target && owner.grab_state >= GRAB_AGGRESSIVE)
 		if(!IS_BLOODSUCKER(feed_target) && !IS_VASSAL(feed_target) && !IS_MONSTERHUNTER(feed_target))
-			feed_target.Unconscious((5 SECONDS + level_current))
+			feed_target.Unconscious((5 + level_current) SECONDS)
```
Comment (JohnFulpWillard) [inline fulp_modules/features/antagonists/bloodsuckers/code/powers/feed.dm:97]: I've recently tried to balance things around Bloodsuckers being equal in fighting at level 5 (so ~40 minutes into the game).

At this point;
- It's 10 seconds worth of sleep, at 25 units of blood (in an aggro grab)
- ``SSprocessing`` having a wait of 1 second, This 10 seconds will be 250 units of blood drained.

This leaves the person waking up with 310 cl of blood left (55% of your blood), which gives you about 10 lifeticks of breaking out of the grab, getting out of wherever the Bloodsucker has you trapped, and getting help, until you fall asleep of blood loss.

Do we want Feed to be this powerful?

File: fulp_modules/features/antagonists/bloodsuckers/code/powers/feed.dm (hunk 3 above)
Comment (tattax) [reply fulp_modules/features/antagonists/bloodsuckers/code/powers/feed.dm:98]: look at the line above, else statement has the same effect

**Pull request merged**